superset/security.py

```diff
@@ -428,3 +429,10 @@ def set_perm(self, mapper, connection, target):  # noqa
                     view_menu_id=view_menu.id,
                 ),
             )
+
+    def check_datasource_permission(self, datasource, user=None):
```

Comment:
👍 nice refactor, tiny NIT where the name `assert_datasource_permission ` could be a bit more clear about the fact that it's a "raise if that's not the case". `check_%` could sound more like it's returning a boolean/

Reply:
👍